build-images.sh

```diff
@@ -23,7 +23,7 @@ build_image() {
 
     # Check if the last layer of the base image exists in the published one
     if [[ $IMAGE_INSPECT != *$BASE_IMAGE_LAST_LAYER_ID* ]]  || [[ $FORCE_UPDATE_VERSION == *$VERSION* ]]; then
-     ./download-release.sh
+      ARTIFACT=linux ./download-release.sh
```

Comment:
this isn't the most robust way to access this script. is `.` assumed to be the director where this script is? Maybe define that using the `D=` conversion we have in various scripts.